external-crates/move/crates/move-bytecode-viewer/src/viewer.rs

```diff
@@ -4,17 +4,18 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #![forbid(unsafe_code)]
+use ratatui::{
```

Comment:
```suggestion

use ratatui::{
```
Unless fmt is doing that